packages/office-ui-fabric-react/src/components/Coachmark/examples/Coachmark.Basic.Example.tsx

```diff
@@ -97,7 +102,14 @@ export class CoachmarkBasicExample extends BaseComponent<{}, ICoachmarkBasicExam
             ]}
             onChange={this._onDropdownChange}
           />
-        </div>
+          <Toggle
```

Comment:
👍 for the feature, but do we gain much by adding this example? It feels like this might confuse the a11y story for this example; would be nice to make sure the example only shows the recommended usage.

Reply:
i'm good either way if we want to remove it.

Reply:
I think the example is helpful - the visual treatment being applied with the toggle clarifies what this newly added boolean is for.